frontend/sportsmatch-app/src/components/LoginComponent.tsx

```diff
@@ -0,0 +1,48 @@
+import '../styles/LoginComponent.css'
+import { FaMailBulk, FaLock, FaGoogle, FaFacebook } from 'react-icons/fa'
+
+function LoginComponent() {
+  return (
+    <div className="wrapper">
+      <form action="">
+        <h1>Log in</h1>
+        <div className="input-box">
+          <label htmlFor="email"></label>
+          <input type="email" placeholder="E-mail addres" required />
+          <FaMailBulk className="icon" />
+        </div>
+        <div className="input-box">
+          <label htmlFor="password"></label>
+          <input type="password" placeholder="password" required />
+          <FaLock className="icon" />
+        </div>
+        <div className="remember-forgot">
+          <label>
+            <input type="checkbox" />
+            Remember me{' '}
+          </label>
+          <a href="#">Forgot password</a>
+        </div>
+
+        <button type="submit">Log in</button>
+        <div className="register-link">
+          <p>
+            Dont have an account <a href="signup">Register</a>
+          </p>
+        </div>
+        <p className="alt-login-text">Or log in using</p>
+
+        <div className="alt-login">
+          <div className="facebook">
+            <FaFacebook className="fb-icon" />
```

Comment:
I don't know about if we will implement this functionality, but we can keep it for now and comment it in case we won't use it